src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -25,6 +25,7 @@
   @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE) ExpandableField<Application> application;
   Long applicationFeeAmount;
   Long canceledAt;
+  String cancellationReason;
```

Comment:
Deleted the previous comment because i posted the internal link.. 
Should we add 'review' field? It seems to be available in OpenAPI, and our source code